packages/block-editor/src/components/inner-blocks/use-nested-settings-update.js

```diff
@@ -4,7 +4,7 @@
 import { useLayoutEffect, useState } from '@wordpress/element';
 import { useRegistry } from '@wordpress/data';
 import deprecated from '@wordpress/deprecated';
-import isShallowEqual from '@wordpress/is-shallow-equal';
+import { isShallowEqual } from '@wordpress/is-shallow-equal';
```

Comment:
Adding `isShallowEqual` as a named export in the package, in addition to the existing default export, and using the named export consistently, this all helps a lot with CJS/ESM reliability.

When an ESM module `imports` a CJS module, it works, because both Node.js and bundlers have interoperability, but `default` exports are interpreted differently in some cases. Typically when a module has _both_ default and named exports. That's why we should avoid this "default+named" layout.